projects/plugins/jetpack/_inc/client/sharing/publicize.jsx

```diff
@@ -71,12 +88,56 @@ export const Publicize = withModuleSettingsFormHelpers(
 								link: getRedirectUrl( 'jetpack-support-publicize' ),
 							} }
 						>
-							<p>
-								{ __(
-									'Connect your website to the social media networks you use and share your content across all your social accounts with a single click. When you publish a post, it will appear on all connected accounts.',
-									'jetpack'
-								) }
-							</p>
+							<p>{ jetpackSocialText }</p>
+							{ showUpgradeLink && (
+								<React.Fragment>
+									<p>
+										{ ! hasSocialBasicFeatures
+											? createInterpolateElement(
+													__(
+														'<moreInfo>Upgrade to a Jetpack Social plan</moreInfo> to get unlimited shares and advanced media sharing options.',
+														'jetpack'
+													),
+													{
+														moreInfo: (
+															<a
+																href={ getRedirectUrl(
```

Comment:
Since these links link outside of wp-admin, I would recommend using ExternalLink here.

Reply:
I thought of using the `ExternalLink`, but it doesn't accept the `target` param, which means the link always opens in a new tab and I think case we would like for it to open n the same. Do you think we can do anything else to work around that?